aptos-move/framework/move-stdlib/sources/Hash.move

```diff
@@ -3,6 +3,7 @@
 /// The functions in this module are natively declared both in the Move runtime
 /// as in the Move prover's prelude.
 module Std::Hash {
+    native public fun sip_hash<MoveValue>(v: &MoveValue): u64;
```

Comment:
haha, why, why? it might be great to leave a note in here on what motivated you down this path -- probably in the BucketTable itself